localization/english/religion/wc_religion_l_english.yml

```diff
@@ -5,4 +5,502 @@
+ 
+ gurubism:0 "Gur'atali"
+ gurubism_adj:0 "Gur'atali"
+ gurubism_adherent:0 "Gur'atali"
+ gurubism_adherent_plural:0 "Gur'atali"
+ gurubism_desc:0 "A pantheon of Loa founded by the Gurubashi trolls. The concept of Death occupies a significant place in this pantheon together with Bwonsamdi, Loa of Death. $gurubism_adherent_plural$ pay special attention to interactions with the body of the deceased like ritual cannibalism."
+ 
+ drakkism:0 "Drak'atali"
+ drakkism_adj:0 "Drak'atali"
+ drakkism_adherent:0 "Drak'atali"
+ drakkism_adherent_plural:0 "Drak'atali"
+ drakkism_desc:0 "A pantheon of Loa founded by the Drakkari trolls. Living in harsh conditions requires a strong body and a keen mind. The Drakkari spend their entire lives making stronger and smarter as their Loa demand."
+ 
+ amism:0 "Am'atali"
+ amism_adj:0 "Am'atali"
+ amism_adherent:0 "Am'atali"
+ amism_adherent_plural:0 "Am'atali"
+ amism_desc:0 "A pantheon of Loa founded by the Amani trolls. Hunting is sacred for the Amali, so the Loa of the Hunt are revered more than others. They don't care who they hunt: an animal or a person. However, it is considered more honorable to eat your opponent than some small animal."
+ 
+ farrism:0 "Far'atali"
+ farrism_adj:0 "Far'atali"
+ farrism_adherent:0 "Far'atali"
+ farrism_adherent_plural:0 "Far'atali"
+ farrism_desc:0 "A pantheon of Loa founded by the Farraki trolls. Balancing on the brink of heresy, they perform bloody rituals reminiscent of those practiced by Hakkari. Their main mystical Loa, Mueh'zala, is unknown to other pantheons."
+ 
+ cult_of_hakkar:0 "Hakkari"
+ cult_of_hakkar_adj:0 "Hakkari"
+ cult_of_hakkar_adherent:0 "Hakkari"
+ cult_of_hakkar_adherent_plural:0 "Hakkari"
+ cult_of_hakkar_desc:0 "Hakkar the Soulflayer, a vile, bloodthirsty god, brought the trolls great power, but in return required souls to be sacrificed to him. Known in the Loa pantheons as the Loa of Blood, worshiping him was declared illegal in all other confessions."
+ 
+ cult_of_sethraliss:0 "Sethrali"
+ cult_of_sethraliss_adj:0 "Sethrali"
+ cult_of_sethraliss_adherent:0 "Sethrali"
+ cult_of_sethraliss_adherent_plural:0 "Sethrali"
+ cult_of_sethraliss_desc:0 "A pantheon of Loa founded by the sethrak. Sethraliss, a snake loa, is the main Loa of this pantheon. She treats her followers like a loving mother to her children. Legends tell that she once sacrificed herself to save Vol'dun and its inhabitants and no doubt she will do it again when she's reborn."
+ 
+ cult_of_ariwo:0 "Ariwi"
+ cult_of_ariwo_adj:0 "Ariwi"
+ cult_of_ariwo_adherent:0 "Ariwi"
+ cult_of_ariwo_adherent_plural:0 "Ariwi"
+ cult_of_ariwo_desc:0 "A pantheon of Loa founded by the vulpera. Ariwo, a fox loa, is the main Loa of this pantheon. He teached his children to stick together and be kind to each other because this is the only way to survive. Despite the fact that Ariwo disappeared thousands of years ago, his followers still honor his covenants."
```

Comment:
```suggestion
 cult_of_ariwo_desc:0 "A pantheon of Loa founded by the vulpera. Ariwo, a fox loa, is the main Loa of this pantheon. He taught his children to stick together and be kind to each other because he felt it was the only way to survive. Despite the fact that Ariwo disappeared thousands of years ago, his followers still honor his covenants."
```